examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -1439,6 +1445,49 @@ impl FrontendNode {
         Ok(())
     }
 
+    fn test_channel_label(&mut self) -> TestResult {
+        let hash = vec![1, 2, 3, 4];
+        let label = Label {
+            confidentiality_tags: vec![oak_abi::label::web_assembly_module_tag(&hash)],
+            integrity_tags: vec![],
+        };
+        let (out_handle, in_handle) = oak::channel_create_with_label(&label).unwrap();
+        let expected = Ok(label);
+        expect_eq!(expected, oak::channel_label(in_handle.handle));
+        expect_eq!(expected, oak::channel_label(out_handle.handle));
```

Comment:
super-nit: most of the tests have a blank line to separate test code from test cleanup code.